pkg/condition/condition.go

```diff
@@ -20,6 +20,15 @@ func AddOrUpdateStatusConditions(conditions []toolchainv1alpha1.Condition, newCo
 	return conditions, atLeastOneUpdated
 }
 
+func FindConditionByType(conditions []toolchainv1alpha1.Condition, conditionType toolchainv1alpha1.ConditionType) toolchainv1alpha1.Condition {
```

Comment:
please add a test for this function ;)

Reply:
@xcoulon thanks, will add :+1:

Reply:
Done https://github.com/codeready-toolchain/toolchain-common/pull/36/commits/a247e08b1bed03a3437043ae547c370912ad3eaa